concourse/steps/release.py

```diff
@@ -390,17 +407,49 @@ def validate(self):
     def apply(
         self,
     ):
+        # clean repository if required
+        worktree_dirty = bool(self.github_helper._changed_file_paths())
```

Comment:
During test I get the error message:
```
WARNING: An error occured while applying step 'Create Next Cycle Commit': 'GitHubRepositoryHelper' object has no attribute '_changed_file_paths'
Traceback (most recent call last):
  File "<string>", line 128, in execute
  File "<string>", line 415, in apply
AttributeError: 'GitHubRepositoryHelper' object has no attribute '_changed_file_paths'
```

Reply:
Yup, was related to the `git_helper` `github_helper` issue :sweat_smile: